src/app/components/Onboard/Shared/steps.jsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+export default function Steps(props) {
```

Comment:
Should style be used if we're using tailwind classes? From what I can grok that could make adding padding difficult if classname is passed down unless some logic is written to check for the existing spacing classes on the component.

Instead it may work better to pass down some defined props like "size", "color", "spacing" etc to build the list of classes. Overall though I think this level of customisation of a component with classname is not worth it, static component styes, a flexible grid system and I think that should be it.